app/views/all_resources/index.html.erb

```diff
@@ -1,44 +1,47 @@
-<section class='row'>
-  <div class="col-md-6 justify-content-start">
-    <h2>All <%= @playlist_videos.current_video.category.capitalize %> Videos</h2>
-    <div id="player">
-      <script src="https://www.youtube.com/player_api" ></script>
-      <script>
-        let player;
-        let position;
-        function onYouTubePlayerAPIReady() {
-          player = new YT.Player('player', {
-            width: '100%',
-            height: '390px',
-            videoId: '<%= @playlist_videos.current_video.video_id %>',
-            events: {
-              onReady: onPlayerReady,
-            }
-          });
-        }
-
-        function onPlayerReady(event) {
-          event.target.playVideo();
-        }
-      </script>
+<section class='container'>
+  <section class='row content'>
+    <div class="col-md-2">
     </div>
-    <div class='video-links'>
-      <% @playlist_videos.videos.each do |video| %>
-        <li><%= link_to video.title, "/all_resources?video_id=#{video.video_id}&category=#{video.category}" %></li>
-      <% end %>
+    <div class="col-md-6">
+      <h2>All <%= @playlist_videos.current_video.category.capitalize %> Videos</h2>
+      <div id="player">
+        <script src="https://www.youtube.com/player_api" ></script>
+        <script>
+          let player;
+          let position;
+          function onYouTubePlayerAPIReady() {
+            player = new YT.Player('player', {
+              width: '100%',
+              height: '390px',
+              videoId: '<%= @playlist_videos.current_video.video_id %>'
```

Comment:
I like how you removed autoplay

Reply:
Me too! Getting the additional links to open in a new tab would be a nice touch too!